lib/cisco_node_utils/cmd_ref/snmp_notification_receiver.yaml

```diff
@@ -0,0 +1,50 @@
+# snmp_notification_receiver
+---
+port:
+  config_get: "show running-config snmp all"
+  config_get_token: '/^snmp-server host %s.*version.* udp-port (\d+).*$/'
```

Comment:
For future work it would be preferable to use named keys in the config_get_token as well (`'/^snmp-server host <ip>.*version.* udp-port (\d+).*$/'`) but this isn't critical at this time.